src/NBitcoin/Networks.cs

```diff
@@ -170,7 +170,7 @@ private static Network InitTest()
                 RootFolderName = BitcoinRootFolderName,
                 DefaultConfigFilename = BitcoinDefaultConfigFilename,
                 Magic = 0x0709110B,
-                alertPubKeyArray = Encoders.Hex.DecodeData("04302390343f91cc401d56d68b123028bf52e5fca1939df127f63c6467cdf9c8e2c14b61104cf817d0b780da337893ecc4aaff1309e536162dabbdb45200ca2b0a"),
```

Comment:
I can't see the alertPubKey setting in all the other networks constructors, is it on purpose ?

Reply:
yeah only bitcoin main and bitcoin test have it.

Reply:
I think its deprecated in bitcoin core so not really needed.